src/middleware/createCsrfErrorMiddleware.ts

```diff
@@ -0,0 +1,21 @@
+import { NextFunction, Request, Response } from "express";
+import { CsrfError } from "@companieshouse/web-security-node";
+
+const csrfErrorTemplateName = "403";
+
+const csrfErrorHandler = (
+  err: CsrfError | Error,
```

Comment:
should we include `| null | undefined` here? I know TS won't allow it, but once transpiled into JS, could a `null` or `undefined` could potentially come in? Same for other middleware I guess...